### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.0.5

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
 
+<!-- 8.0.5 START -->
+
+# 8.0.5 (2021-07-08)
+
+### Features and enhancements
```
Comment (oddlittlebird) [inline CHANGELOG.md:6]:
```suggestion
## Features and enhancements
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+
+* **Cloudwatch Logs:** Send error down to client. [#36277](https://github.com/grafana/grafana/pull/36277), [@zoltanbedi](https://github.com/zoltanbedi)
+* **Folders:** Return 409 Conflict status when folder already exists. [#36429](https://github.com/grafana/grafana/pull/36429), [@dsotirakis](https://github.com/dsotirakis)
+* **TimeSeries:** Do not show series in tooltip if it's hidden in the viz. [#36353](https://github.com/grafana/grafana/pull/36353), [@dprokop](https://github.com/dprokop)
+
+### Bug fixes
```
Comment (oddlittlebird) [inline CHANGELOG.md:12]:
```suggestion
## Bug fixes
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+* **DashboardQueryRunner:** Fixes unrestrained subscriptions being created. [#36371](https://github.com/grafana/grafana/pull/36371), [@hugohaggmark](https://github.com/hugohaggmark)
+* **DateFormats:** Fix reading correct setting key for use_browser_locale. [#36428](https://github.com/grafana/grafana/pull/36428), [@torkelo](https://github.com/torkelo)
+* **Links:** Fix links to other apps outside Grafana when under sub path. [#36498](https://github.com/grafana/grafana/pull/36498), [@torkelo](https://github.com/torkelo)
+* **Snapshots:** Fix snapshot absolute time range issue. [#36350](https://github.com/grafana/grafana/pull/36350), [@torkelo](https://github.com/torkelo)
+* **Table:** Fix data link color. [#36446](https://github.com/grafana/grafana/pull/36446), [@tharun208](https://github.com/tharun208)
+* **Time Series:** Fix X-axis time format when tick increment is larger than a year. [#36335](https://github.com/grafana/grafana/pull/36335), [@torkelo](https://github.com/torkelo)
+* **Tooltip Plugin:** Prevent Tooltip render if field is undefined. [#36260](https://github.com/grafana/grafana/pull/36260), [@ashharrison90](https://github.com/ashharrison90)
```
Comment (oddlittlebird) [inline CHANGELOG.md:22]:
```suggestion
* **Tooltip Plugin:** Prevent tooltip render if field is undefined. [#36260](https://github.com/grafana/grafana/pull/36260), [@ashharrison90](https://github.com/ashharrison90)
```

**Review by oddlittlebird — APPROVED** [review]
A few minor edits, then g2g

File: CHANGELOG.md (hunk 1 above)
Comment (tskarhed) [reply CHANGELOG.md:6]: We have level 3 headings for these throughout the changelog. Is this something that we want to change?

Comment (oddlittlebird) [reply CHANGELOG.md:6]: Yes. It should go from h1 to h2, not h1 to h3.

File: docs/sources/release-notes/release-notes-8-0-5.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+title = "Release notes for Grafana 8.0.5"
+[_build]
+list = false
++++
+
+<!-- Auto generated by update changelog github action -->
+
+# Release notes for Grafana 8.0.5
+
+### Features and enhancements
```
Comment (oddlittlebird) [inline docs/sources/release-notes/release-notes-8-0-5.md:11]: All these h3s should be h2s.

File: CHANGELOG.md (hunk 1 above)
Comment (tskarhed) [reply CHANGELOG.md:6]: Makes sense! We're going to merge this PR and finish up the release first though. Then we'll also have a look at updating Grot so it generates the correct headings, and fix the current ones.

**Pull request merged**